### PR TITLE
Smart menu

### PR DESCRIPTION
**Pull request by Worll**

### CODE REVIEW

File: app/(admin)/admin/dashboard/navigation/page.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { createServerComponentClient } from "@supabase/auth-helpers-nextjs";
+import { cookies } from "next/headers";
+import NavigationForm from "@/components/admin/NavigationForm";
+
+export default async function Page() {
+  const supabase = createServerComponentClient({ cookies });
+
+  const { data: navigation } = await supabase.from("SmartMenu").select("*");
+  console.log(navigation);
```
Comment (Neutronas) [inline app/(admin)/admin/dashboard/navigation/page.tsx]: Remove console logs and comment.

**Review by Neutronas — CHANGES_REQUESTED** [review]

File: app/(admin)/admin/dashboard/navigation/page.tsx (hunk 1 above)
Comment (Worll) [reply app/(admin)/admin/dashboard/navigation/page.tsx:9]: Nebesvaik viskas cia gerai

Comment (Worll) [reply app/(admin)/admin/dashboard/navigation/page.tsx]: Everything is okay here mate

Comment (Neutronas) [reply app/(admin)/admin/dashboard/navigation/page.tsx]: It's not. Please keep production code clean.

Comment (Worll) [reply app/(admin)/admin/dashboard/navigation/page.tsx]: Fixed it my sugar boo

**Review by Neutronas — APPROVED** [review]

**Pull request merged**